RecoverVm/readme.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+Occasionally Azure IaaS VMs may not start because there is something wrong with the operating system (OS) disk preventing it from booting up correctly.
+In such cases it is a common practice to recover the problem VM by performing the following steps:
+
+- Delete the VM (but keep the disks)
+- Attach the disk(s) to another bootable VM as a Data Disk
+- Run the script https://github.com/sebdau/azpstools/blob/master/FixDisk/TS_RecoveryWorker2.ps1 as an elevated administrator from the recovery VM
```

Comment:
just make a relative path.  Should be [TS_RecoveryWorker2.ps1](./TS_RecoveryWorker2.ps1)